compiler/plugins/target/ROCM/Dialect/ROCM/IR/ROCMDialect.cpp

```diff
@@ -62,4 +63,23 @@ SmallVector<StringRef> ROCMDialect::getBuiltinNames() {
   return llvm::to_vector(builtins.getMap().keys());
 }
 
+const ArrayRef<Util::FuncOp> ROCMDialect::getMlirUKernels() {
+  std::lock_guard<std::mutex> guard(mlirUkernelsMutex);
+  if (mlirUkernels.empty()) {
+    const iree_file_toc_t *toc = iree_mlir_ukernels_amdgpu_create();
+    llvm::SmallVector<ModuleOp> result;
+    for (size_t i = 0, e = iree_mlir_ukernels_amdgpu_size(); i != e; ++i) {
+      auto module = this->getOrLoadBuiltinModule(toc[i].name);
```

Comment:
nit: module is a keyword in c++20
```suggestion
      ModuleOp moduleOp = this->getOrLoadBuiltinModule(toc[i].name);
```